reviewing/plugin-commands-licenses/src/outputRenderer.ts

```diff
@@ -112,13 +112,45 @@ function renderLicensesTable (
   for (let i = 0; i < columnNames.length; i++)
     columnNames[i] = chalk.blueBright(columnNames[i])
 
+  const data = [
+    columnNames,
+    ...deduplicateLicensesPackages(sortLicensesPackages(licensePackages))
+      .map((licensePkg) => columnFns.map((fn) => fn(licensePkg))),
+  ]
+  let detailsColumnMaxWidth = 40
+  let packageColumnMaxWidth = 0
+  let licenseColumnMaxWidth = 0
+  if (opts.long) {
+    // Use the package link to determine the width of the details column
+    detailsColumnMaxWidth = licensePackages.reduce((max, pkg) => Math.max(max, pkg.homepage?.length ?? 0), 0)
+    for (let i = 1; i < data.length; i++) {
+      const row = data[i]
+      const texts = row[2].split('\n')
+      const repeatCount = Math.max(0, texts.length - 1)
+      row[0] += '\n '.repeat(repeatCount) // Add extra spaces to the package column
+      row[1] += '\n '.repeat(repeatCount) // Add extra spaces to the license column
+      packageColumnMaxWidth = Math.max(packageColumnMaxWidth, row[0].length)
+      licenseColumnMaxWidth = Math.max(licenseColumnMaxWidth, row[1].length)
+    }
+    const remainColumnWidth = process.stdout.columns - packageColumnMaxWidth - licenseColumnMaxWidth - 20
+    if (detailsColumnMaxWidth > remainColumnWidth) {
+      detailsColumnMaxWidth = remainColumnWidth
+    }
+    detailsColumnMaxWidth = Math.max(detailsColumnMaxWidth, 40)
```

Comment:
Another question, this still occurs when the terminal window is small, how can I avoid the error? 🤔